sdk/storage/storage-blob/swagger/README.md

```diff
@@ -28,6 +28,24 @@ package-version: 12.7.0-beta.1
 See the [AutoRest samples](https://github.com/Azure/autorest/tree/master/Samples/3b-custom-transformations)
 for more about how we're customizing things.
 
+```yaml
```

Comment:
add comments for the transform so we know what it is for

Reply:
Added